src/components/orgunits/styles/OrgUnitSelect.module.css

```diff
@@ -0,0 +1,19 @@
+.orgUnitSelect :global(.orgUnitTreeWrapper) {
+    max-height: 220px;
+}
+
+.orgUnitSelect :global(.userOrgUnitsWrapper label) {
+    margin-right: 34px;
+}
+
+.event :global(.orgUnitTreeWrapper),
```

Comment:
why do these have a larger max-height?

Reply:
I'm adjusting the size of the org unit tree to avoid a vertical scroll in the layer dialog that would hide some of the selects. Some layer dialogs have more selects than others as shown in the screenshots above.